R/fotmob_stats.R

```diff
@@ -326,7 +262,8 @@ fotmob_get_season_stats <- function(
     cached = TRUE
 ) {
 
-  match.arg(team_or_player, several.ok = FALSE)
+  stopifnot('Must specify one of either `"team"` or `"player"` for `team_or_player`' = length(team_or_player) == 1)
```

Comment:
drive by fix for a bug. prior logic didn't actually raise an error here if a user specified both team and player